src/markerStyle.ts

```diff
@@ -0,0 +1,156 @@
+import * as vscode from 'vscode'
+
+interface Config {
+  markerStyle?: string
+  markerWholeLine?: boolean
+  markerColor?: string
+  markerJSON?: MyDecorationRenderOptions // will auto generate
+  markerJSONCustom?: MyDecorationRenderOptions
+}
+interface Edit {
+  line: number
+  character: number
+  filepath: string
+  filename: string
+  lineText: string
+}
+
+let globalConfig: Config = {}
+let context: vscode.ExtensionContext
```

Comment:
It seems unnecessary to hold a reference to context like this. It could be captured in the triggerDecorations function instead.

Reply:
the variable：context is used for differrent functions